cmd/flow-collector/main.go

```diff
@@ -445,7 +474,7 @@ func main() {
 	log.Printf("COLLECTOR: server listening on %s", addr)
 	s := &http.Server{
 		Addr:    addr,
-		Handler: mux,
+		Handler: enableGzipMiddleware(mux),
```

Comment:
@bartoval I believe this could be simplified by using: `Handler: handlers.CompressHandler(mux)` instead.
You'd also need this extra import: `"github.com/gorilla/handlers"`.

Let me know if it works for you.

Reply:
Thank you for your suggestion @fgiorgetti . Much better now 🕺